pyproject.toml

```diff
@@ -32,12 +32,14 @@ h5py = {version ="^3.7.0", optional = true}
 wandb = {version ="^0.13.4", optional = true}
 tensorflow = {version ="2.10.0", optional = true}
 ipykernel = {version = "*", optional = true}
+seaborn = {version = '^0.12.2', optional = true}
 
 [tool.poetry.extras]
 docs = ["Sphinx", "sphinx-autodoc-typehints", "sphinx-mdinclude", "sphinx-rtd-theme"]
 dev = ["pytest", "pytest-cov", "prospector", "pyroma", "h5py"]
 rnn = ["wandb", "tensorflow"]
 tutorials = ["ipykernel"]
+visuals = ["seaborn"]
```

Comment:
Do you think we should add pandas to visuals? It is used in ```plot_pdf`` function.

Reply:
I think so, although pandas got installed automatically when installing seaborn.